sdk/spring/azure-spring-boot-starter-active-directory-b2c/CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 # Release History
 
 ## 3.5.0-beta.1 (Unreleased)
+### Key Bug Fixes
+- Fix AAD B2C cannot complete OpenID Connect metadata document acquisition through issuer.
```

Comment:
Can we add link to related Issue or PR?
Just like this one: https://github.com/Azure/azure-sdk-for-java/blob/2ea8f94fb8523dbf4a6ee5961223df599a063620/sdk/spring/azure-spring-boot-starter-active-directory/CHANGELOG.md#key-bug-fixes